start_cerberus.py

```diff
@@ -119,6 +126,16 @@ def main(cfg):
                 for namespace, failures in failed_pods_components.items():
                     logging.info("%s: %s", namespace, failures)
 
+            for namespace in failed_pods_components.keys():
+                dir_name = "reports/" + namespace + '-logs'
+                if os.path.isdir(dir_name):
+                    runcommand.invoke("rm -R " + dir_name)
+                    print("Deleted existing", dir_name, 'directory')
+                log_status = runcommand.invoke("oc adm inspect "
```

Comment:
Can we change the variable name to something like command_out?

Reply:
Sure.